torch/_torch_docs.py

```diff
@@ -435,6 +435,13 @@ def merge_dicts(*dicts):
             [ 0.7573, -3.9555, -2.8681]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.adjoint,
```

Comment:
For consistency, the method variant typically points to the function variant and it'd be nice to maintain that here to save people searching for this op a click

Reply:
From a user's perspective, mH the property is equivalent to calling the function adjoint, right? I'm not sure I'd bother describing one as an alias of another

Reply:
What are you suggesting in the second message?

Reply:
For .mH it could say something straightforward like "Accessing this property is equivalent to calling torch.adjoint(t)."